core/ViewDataTable/Config.php

```diff
@@ -533,6 +534,16 @@ public function __construct()
         foreach ($this->selectable_periods as $period) {
             $this->translations[$period] = ucfirst(Piwik::translate('Intl_Period' . ucfirst($period)));
         }
+
+        foreach (Dimension::getAllDimensions() as $dimension) {
```

Comment:
Any chance we maybe only add them in the `HtmlTable` config? Other visualisations shouldn't need it? or even better only in the  HtmlTable visualisation and only for the needed dimensions re performance?